conf.d/openstack.yaml.example

```diff
@@ -62,6 +62,15 @@ init_config:
       # need to set to False when using self-signed certs
       # ssl_verify: true
 
+      #IDs of Projects (aka Tenants) to monitor
+      project_ids:
+          - 2f60042a0d464023905aaebf23a350ff
```

Comment:
This one is my bad. That should probably be commented out like all the other examples.